### DISCUSSION TITLE
'Save unpacked image' does nothing, possibly incomplete and 1 other issue

### DISCUSSION THREAD

**Discussion by 8bit-coder** [vgmtrans/vgmtrans#222] [Q&A] [answered]
This is an awesome program but there's two issues I'd like to bring attention to:
Save unpacked image: It seems to be a tool to just extract everything to a folder but it doesn't really do anything at the moment. Am I using it wrong or is it simply incomplete?

Appveyor has an artifact retention of 6 months, so as of right now, there are no downloadable builds in the appveyor page because all of them have been discarded since they are older than 6 months. Is there anything I can do to get the latest version and not be stuck with the 2017 version?

**Comment by sykhro** [comment, marked as answer]
> Save unpacked image: It seems to be a tool to just extract everything to a folder 

It's a misnomer: it saves the "raw" files, which may or may not be identical to the files passed in to be analyzed (e.g. when providing PSX files, VGMTrans extracts the VABs to process them; you can use that option to save them).

> Is there anything I can do to get the latest version and not be stuck with the 2017 version?

This aspect should be a bit better now; a proper release is due soon
